package.json

```
@@ -31,16 +31,12 @@
       {
         "id": "zndsl",
         "extensions": [
-          ".feature",
           ".zndsl"
         ],
         "aliases": [
           "ZombieNet"
         ]
       }
-    ],
-    "files.associations": {
-      "*.feature": "zndsl"
-    }
+    ]
```

Don't we need to keep the pattern ?

```suggestion
    ],
    "files.associations": {
      "*.zndsl": "zndsl"
    }
```

---

I'm not sure - but lets keep it for now